codespell_lib/data/dictionary.txt

```diff
@@ -21367,6 +21368,7 @@ efffectiveness->effectiveness
 efffects->effects
 efffort->effort
 effforts->efforts
+efficacity->efficacy
```

Comment:
It turns out `efficacity` is a valid word according to these dictionaries:
* [OED](https://www.oed.com/search/dictionary/?scope=Entries&q=efficacity)
* [Cambridge](https://dictionary.cambridge.org/dictionary/english/efficacity)
* [Collins](https://www.collinsdictionary.com/dictionary/english/efficacity)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/efficacity)

but is not part of these:
* [Longman](https://www.ldoceonline.com/spellcheck/english/?q=efficacity)
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=efficacity)

On the other hand, the Google Ngram Viewer shows that `efficacity` is a few orders of magnitude less frequent than `efficacy`:
https://books.google.com/ngrams/graph?content=efficacity%2Cefficacy&year_start=1800&year_end=2019&corpus=en-2019

As usual, I think it's best to check whether the word is found in the [SCOWL (and friends)](http://app.aspell.net/lookup?dict=en_GB-large;words=efficacity) dictionaries, the reference for open source English spellchecking. It is not. So I'm OK with this change.